libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts

```diff
@@ -103,7 +103,8 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
 
   /**
    * The ARIA label for the dropdown. This sets the dropdown's `aria-label` attribute to provide a text equivalent for screen readers
-   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
+   * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility). If multiple dropdowns are used in proximity to each
```

Comment:
I think this can be more specific. Along the lines of "multiple dropdowns with the same label on the same page, " - key point being it's when used on the same page.

Reply:
I updated my suggested edit to use this recommendation.

Reply:
Resolving this because it was incorporated into a separate comment thread.